config/laravel-console-dusk.php

```diff
@@ -12,4 +12,9 @@
         'screenshots' => storage_path('laravel-console-dusk/screenshots'),
         'log'         => storage_path('laravel-console-dusk/log'),
     ],
+    /*
+    | When false it will show a Chrome window while running. Within production
+    | it will be forced to run in headless mode.
```

Comment:
Just a suggestion, but maybe add a header for consistency.
```suggestion

    /*
    | --------------------------------------------------------------------------
    | Headless Mode
    | --------------------------------------------------------------------------
    |
    | When false it will show a Chrome window while running. Within production
    | it will be forced to run in headless mode.
```